en/flying/terrain_following_holding.md

```diff
@@ -0,0 +1,40 @@
+# Terrain Following/Holding
```

Comment:
@bkueng I agree with you re importance of this feature. So what I have done is added this as a stand alone doc here. I have not yet updated the rangefinder doc to cross link and doubtless this will change as we clarify Range Aid.

This is worth looking at now though. 

My gut feeling is that Range aid is very similar to the terrain hold but applies to more modes - specifically to landing and hold. So it would be worth documenting it in this section too. 
Similarly, range aid provides the same benefits of terrain following to missions, return and similar modes - albeit it does have a user-defined velocity restriction. My leaning therefore is to simply make this clear as an option alongside the other too and make it clear that it will need to be tuned for speed.

Thoughts?

Reply:
> My gut feeling is that Range aid is very similar to the terrain hold but applies to more modes - specifically to landing and hold.

That's also my understanding.

> Similarly, range aid with a large velocity limit will emulate terrain following - but allow them in missions, return and similar modes.

Right, but it was not designed for that. I'm not sure if that has other consequences.

Reply:
I'm sure it will. More generally I wonder if the EKF compensates the altitude for vehicle roll etc. At a certain point I assume the rangefinder might provide meaningless results.